packages/core/frontend/sirius-components-core/src/workbench/Toast.tsx

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import IconButton from '@material-ui/core/IconButton';
+import Snackbar from '@material-ui/core/Snackbar';
+import CloseIcon from '@material-ui/icons/Close';
+import { ToastProps } from './Toast.types';
+
+export const Toast = ({ message, open, onClose }: ToastProps) => {
+  return (
+    <Snackbar
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'right',
+      }}
+      open={open ?? !!message}
+      autoHideDuration={6000}
```

Comment:
This duration is twice as long as the one from before in most (all?) usages of `Snackbar`.
Is it voluntary (e.g. because we expect longer messages to be displayed now) or a leftover from development/debug?

Reply:
It was for testing convenience, I switch back to 3000.